packages/manager/apps/pci-databases-analytics/src/pages/Root.page.tsx

```diff
@@ -1,4 +1,33 @@
+import { redirect } from 'react-router-dom';
+import queryClient from '@/query.client';
 import Services from './services/Services.page';
+import { getServices } from '@/data/api/database/service.api';
+
+interface ServicesProps {
+  params: {
+    projectId: string;
+    category: string;
+  };
+  request: Request;
+}
+export const Loader = ({ params }: ServicesProps) => {
```

Comment:
I don't see how this function is used, is it a react component ? If so the returned type seems weird and it should use hooks instead

Reply:
It's a loader function used by react-router: https://reactrouter.com/start/framework/route-module#loader